src/webargs/core.py

```diff
@@ -1,11 +1,9 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
 import functools
 import inspect
 import logging
 import warnings
 from copy import copy
+from collections.abc import Mapping
 
 try:
     import simplejson as json
```

Comment:
I suspect this could be removed.

Reply:
Removed in b734c5508f63cea93f3f6cee34807f1d400d2fe9!